packages/next/taskfile.js

```diff
@@ -1978,6 +1985,14 @@ export async function lib(task, opts) {
   notify('Compiled lib files')
 }
 
+export async function lib_esm(task, opts) {
```

Comment:
another option to write this part would be to make the lib task a compound task with parallel tasks like

```
function lib(task, opts) {
await task.parallel(
    ['lib_cjs', 'lib_esm', ],
    opts
  )
}
```

This is sort of equivalent but makes it less easy to cleanup so I went for my approach.

thoughts?